lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -73,6 +75,48 @@ deployDebugApk:
         https://slack.com/api/files.upload
   cache: {}
 
+testRelease:
+  stage: test
+  only:
+    - tags
+  script:
+    - ./gradlew -Pci --console=plain testRelease
+  artifacts:
+    when: always
+    reports:
+      junit:
+        - "**/TEST-*.xml"
+
+assembleReleaseApk:
+  stage: assembleRelease
+  only:
+    - tags
+  script:
+    - echo ${KEYSTORE_FILE} | base64 -d > my.keystore
```

Comment:
Can you please add the `KEYSTORE_FILE`, `KEYSTORE_PASSWORD`, `KEY_ALIAS`, and `KEY_PASSWORD` to the `Gradle Android Pipeline` documentation at https://github.com/chesapeaketechnology/gitlab-templates/blob/release/1.3/README.md#customization-1

Reply:
Yes, sure thing!